src/types/accessors/AbstractMetabolicModel.jl

```diff
@@ -141,6 +141,25 @@ groups collect these sets of enzymes for convenient analysis.
 """
 )
 
+@make_variable_semantics(
+    :metabolite_log_concentration,
```

Comment:
can we generalize to log-concentration to anything, or should this stay metabolite specific? (just golfing the name)

Reply:
hmm probably only for metabolites does it make sense. Other concentration like quantities will likely not get logged since they won't feature into thermo calculations e.g. enzyme concentrations